crates/red_knot/src/types/infer.rs

```diff
@@ -148,6 +148,15 @@ fn infer_expr_type(db: &dyn SemanticDb, file_id: FileId, expr: &ast::Expr) -> Qu
             let attr_name = &Name::new(&attr.id);
             value_type.get_member(db, attr_name)
         }
+        ast::Expr::Call(ast::ExprCall { func, .. }) => {
```

Comment:
I don't think we will need this for this lint rule (it doesn't require actually inferring type of an entire call expression). Of course we will need it eventually.

Reply:
I'll remove it. Lmk if you decide I should keep it and I can revert.

Reply:
No I don't think it makes sense to add it in this PR.